### PR TITLE
Update alacritty versions

### PR DESCRIPTION
**Pull request by chrisduerr**
This is only an update to the development versions and does not represent a stable release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,9 @@ The sections should follow the order `Packaging`, `Added`, `Changed`, `Fixed` an
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-## 0.13.0-dev
+## 0.14.0-dev
+
+## 0.13.0
```
Comment (kchibisov) [inline CHANGELOG.md:10]: I think we always used `rc` here? Don't remember.

**Review by kchibisov — CHANGES_REQUESTED** [review]
All the dev bumps except alacritty itself should be non-breaking.

File: CHANGELOG.md (hunk 1 above)
Comment (chrisduerr) [reply CHANGELOG.md:10]: > On master, the version is bumped to `0.3.0-dev` and the `-dev` suffix is removed from the `0.2.0-dev` changelog

Should be fine according to our guideline.

**Comment by chrisduerr** [comment]
> All the dev bumps except alacritty itself should be non-breaking.

Yeah I wasn't actually sure about that. But I guess it's consistent with what we did in the past.

**Review by kchibisov — APPROVED** [review]

**Pull request merged**